### PR TITLE
pre existing condition is not a symptom

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@erikh360 please review

File: higher_health/tests/test_utils.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def get_data(symptoms=0, exposure=False, age="<18"):
         "symptoms_muscles_hurt": "no",
         "symptoms_taste": "no",
         "medical_exposure": "yes" if exposure else "no",
-        "symptoms_pre_existing_condition": "not_sure",
+        "medical_pre_existing_condition": "not_sure",
```
Comment (erikh360) [inline higher_health/tests/test_utils.py]: Dont you want to use `pre_existing_condition` here?

File: higher_health/forms.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ class HealthCheckQuestionnaire(forms.Form):
         widget=forms.RadioSelect,
         required=True,
     )
-    symptoms_pre_existing_condition = forms.ChoiceField(
+    medical_pre_existing_condition = forms.ChoiceField(
```
Comment (erikh360) [inline higher_health/forms.py:119]: Just to confirm, this will make it not count towards the symptom count.

**Review by erikh360 — COMMENTED** [review]
small comment and question

File: higher_health/forms.py (hunk 2 above)
Comment (erikh360) [reply higher_health/forms.py:119]: just saw the ticket, ignore this one

**Pull request merged**